game.html

```diff
@@ -3,21 +3,29 @@
   var answer = Number.parseInt('18');
 
   var guess = Number.parseInt(prompt('Can you guess how many cups of coffee per week it takes to keep me functional without getting jumpy? You have five chances.'));
-  var count=0;
-  while(count < 6)
-    if (guess=== answer){
+  console.log(guess);
+  var count = 0;
+  while(count < 5) {
+
+    if (guess === answer){
         alert('That\'s the perfect amount of coffee!');
       break;
     } else if(guess < answer){
-         guess = Number.parseInt(prompt('So tired! Not enough coffee! Add more.'));
+
+         guess = Number.parseInt(prompt(guess +"?! " + 'That\'s not enough coffee! Add more.'));
+         count = count + 1;
+    }else if(guess > answer){
+         guess = Number.parseInt(prompt(guess +  '?! That\'s way too much coffee. Saying that is just making me feel Jittery! Less is best.'));
          count = count + 1;
-    }else if(guess> answer){
-         guess = Number.parseInt(prompt('Jittery! Too much coffee. Less is best.'));
-         count= count + 1;
     }
- if(count===5){
-   alert("You used all your guesses! You lose.")
-};
+     else if (isNaN(guess)) {
+        guess = prompt('Try entering a number')
+     }
+
+
+  }
+if (count === 5){
+        alert("You used all your guesses! You lose.")
+      }
 })()
 </script>
```

Comment:
Last semantic comment - Be sure to double check all of your indentation. Thinking of those parent : child relationships, everything inside of the <script> tags should be indented. Everything inside the anonymous function should be indented. Everything inside "{  (your code here)  }" should be indented. Etc.